front/app/root.tsx

```diff
@@ -54,11 +60,23 @@ mountApplication();
 
 if (process.env.SENTRY_DSN) {
   import('@sentry/integrations').then((Integrations) => {
-    init({
+    Sentry.init({
       dsn: process.env.SENTRY_DSN,
       environment: process.env.SENTRY_ENV,
       release: process.env.CIRCLE_BUILD_NUM,
-      integrations: [new Integrations.RewriteFrames()],
+      integrations: [
+        new Integrations.RewriteFrames(),
+        new BrowserTracing({
+          routingInstrumentation: Sentry.reactRouterV6Instrumentation(
+            React.useEffect,
+            useLocation,
+            useNavigationType,
+            createRoutesFromChildren,
+            matchRoutes
+          ),
+        }),
+      ],
+      tracesSampleRate: 0.05,
```

Comment:
Set this to 5% - let me know if too high / low.

Reply:
Great, let's start with this and see how we cope :) Should already be enough for some great insights